specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/virtualWan.json

```diff
@@ -2094,6 +2094,77 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/vpnGateways/{gatewayName}/vpnConnections/{connectionName}/vpnLinkConnections/{linkConnectionName}/getikesas": {
```

Comment:
>getikesas [](start = 198, length = 9)

Is there a scenario where an ARM template may need to list the IKE SAS and pass it to another resource? If yes, you should call it listIkeSas instead. (ARM templates allow invoking POST actions called list*.)

Reply:
Hi @majastrz,
No, for ARM template it would only be GET and cannot be used further. I understand that the operation should have been called listikesas to avoid confusion, however, the name getikesas was decided by PM and already deployed from NRP (Yu-Shun Wang: Yu-Shun.Wang@microsoft.com)